src/page-modules/departures/server/journey-planner/journey-gql/stop-place.gql

```diff
@@ -2,6 +2,7 @@ query getStopPlace($id: String!) {
   stopPlace(id: $id) {
     name
     id
+    description
```

Comment:
Is this needed? I couldn't find any use of it

Reply:
The parsing threw an error without it since the type says it's nullable but not optional. Could always have changed the type though, if it's never going to be used.

Reply:
I think this is my fault from a previous PR. If you want you can remove the description from stopPlace as that is not in use or else we can do that in a separate PR 👍

Reply:
Turns out it is used in some places, e.g.
![image](https://github.com/AtB-AS/planner-web/assets/14045515/acedd9e5-0fc5-4c2f-a214-4f70c92d153f)

I say we can leave it.

Reply:
Alright, leave it 👍